### PR TITLE
Updated analytics udfs to views

### PR DESCRIPTION
**Pull request by TJ202**
- this pr contains modifications to sqlpp files in analytics from udfs to views
- tested its working with debug logs on initialising auditor (only func call)

### CODE REVIEW

File: agent_catalog_additional-notices.txt
Comment (glennga) [inline agent_catalog_additional-notices.txt:1]: remove this file from the change (might need to remove this file from your local Git tracking)

File: libs/agentc_core/agentc_core/analytics/ddls/all_sessions.sqlpp [hunk 1]
```diff
@@ -1,73 +1,70 @@
...
-                    rsli.*,
-                    ROW_NUMBER()
-                        OVER (
-                            PARTITION BY
-                                _id
-                            ORDER BY
-                                rsli.timestamp ASC
-                        ) AS rn
-            )
+CREATE OR REPLACE [ANALYTICS?] VIEW Sessions AS
```
Comment (glennga) [inline libs/agentc_core/agentc_core/analytics/ddls/all_sessions.sqlpp:1]: Let's remove the [ANALYTICS?] bits from the queries and replace these with just CREATE OR REPLACE ANALYTICS VIEW (as we'll ultimately have a separate file for the query service UDFs). This means there needs to be updates to how these view DDLs are rendered.

File: libs/agentc_core/agentc_core/analytics/ddls/exchanges.sqlpp [hunk 2]
```diff
@@ -1,88 +1,85 @@
 -- Note: all_sessions.sqlpp should be run before this script.
 -- TODO (GLENN): The purpose of this view is to aid in using Ragas, but we should use the raw_logs instead.
-CREATE OR REPLACE [ANALYTICS?] FUNCTION
```
Comment (glennga) [inline libs/agentc_core/agentc_core/analytics/ddls/exchanges.sqlpp:3]: replace the implementation of this view with the one in the notebook here: https://github.com/couchbaselabs/agent-catalog/blob/master/templates/agents/with_controlflow/notebook.ipynb

Comment (TJ202) [reply libs/agentc_core/agentc_core/analytics/ddls/exchanges.sqlpp:3]: added this under LastMessage view name - lmk if it needs to be named something else

**Comment by glennga** [comment]
LGTM

**Review by glennga — APPROVED** [review]
*LGTM

**Pull request merged**